server/src/config/constants.js

```diff
@@ -1,10 +1,8 @@
+export const LOCATIONS_URI = "http://www.yaledining.org/fasttrack/locations.cfm?version=3";
```

Comment:
Remove the query string :)

Reply:
Why don't we want this to be in constants?

Reply:
I meant the `?version=3` at the end, since that should be handled by `queryBuilder`